inst/include/tthread/tinythread.h

```diff
@@ -866,12 +866,17 @@ inline void * thread::wrapper_function(void * aArg)
   {
     // Uncaught exceptions will terminate the application (default behavior
     // according to C++11)
+
     std::terminate();
   }
 
   // The thread is no longer executing
+  // Originally it executes on all platforms, but I don't think it's right to
+  // call with pthreads, so only make it work on Windows
+#if defined(_TTHREAD_WIN32_)
   lock_guard<mutex> guard(ti->mThread->mDataMutex);
   ti->mThread->mNotAThread = true;
```

Comment:
I think the name is a bit confusing but `mNotAThread` really tracks whether the thread is currently executing or not, so given that intended use I think this code still makes sense.